examples/notebooks/intro/dpk_intro_1_ray.ipynb

```diff
@@ -504,7 +504,8 @@
         "    \"data_files_to_use\": ast.literal_eval(\"['.pdf']\"),\n",
         "    # orchestrator\n",
         "    \"runtime_worker_options\": ParamsUtils.convert_to_ast(worker_options),\n",
-        "    \"runtime_num_workers\": MY_CONFIG.RAY_RUNTIME_WORKERS,\n",
+        "    # \"runtime_num_workers\": MY_CONFIG.RAY_RUNTIME_WORKERS,\n",
```

Comment:
Please decide on using 1 or MY_CONFIG.RAY_RUNTIME_WORKERS and remove commented line. Thanks